$NetBSD: patch-ae,v 1.10 2010/09/24 08:58:41 wiz Exp $

Instead of random text that may or may not make compilers error
out, use #warning.

--- pngconf.h.orig	2009-05-07 16:05:28.000000000 +0000
+++ pngconf.h
@@ -332,8 +332,8 @@
        /* If you encounter a compiler error here, see the explanation
         * near the end of INSTALL.
         */
-           __pngconf.h__ in libpng already includes setjmp.h;
-           __dont__ include it again.;
+#warning           __pngconf.h__ in libpng already includes setjmp.h;
+#warning           __dont__ include it again.;
 #      endif
 #    endif /* __linux__ */
 #  endif /* PNG_SKIP_SETJMP_CHECK */
